realm/realm-annotations-processor/src/test/java/io/realm/processor/RealmProcessorTest.java

```diff
@@ -364,7 +364,7 @@ public void compileRequiredTypes() throws IOException {
     public void compileInvalidRequiredTypes() throws IOException {
         final String[] invalidRequiredAnnotationFieldTypes = {"byte", "short", "int", "long", "float", "double",
                 "boolean", "RealmList<Simple>", "Simple", "Mixed", "RealmList<Mixed>", "RealmDictionary<Simple>",
-                "RealmDictionary<Mixed>"};
+                "RealmDictionary<Mixed>", "RealmSet<Simple>", "RealmSet<Mixed>"};
```

Comment:
Does this test pass? I think it should fail unless you add support for `RealmSet<Mixed>` in the annotation processor 🤔

Reply:
Yes, it does